src/nft/components/collection/Card.tsx

```diff
@@ -453,6 +497,52 @@ const InfoContainer = ({ children }: { children: ReactNode }) => {
   )
 }
 
+const TruncatedTextRow = styled(Row)`
+  padding: 2px;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  display: block;
+  overflow: hidden;
+`
+
+interface ViewMyNftDetailsProps {
+  asset: WalletAsset
+}
+
+const ViewMyNftDetails = ({ asset }: ViewMyNftDetailsProps) => {
```

Comment:
Can we rename this ProfileNftDetails or ProfileNftContent? This will help better differentiate from the NftDetails page

Reply:
sg done